src/hyp3_autorift/process.py

```diff
@@ -370,6 +370,7 @@ def process(
     naming_scheme: Literal['ITS_LIVE_OD', 'ITS_LIVE_PROD'] = 'ITS_LIVE_OD',
     publish_bucket: str = '',
     use_static_files: bool = True,
+    frame_id: str = '',
```

Comment:
```suggestion
    frame_id: str | None = None,
```

This will end up as `None` from HyP3 and later,  `None` is hard-coded as the argument passed to `process_slc` when called from `process_snetinel1_slc_isce3`, so I think it makes sense to consider this an optional parameter.